manual/installation.tex

```diff
@@ -324,6 +324,15 @@ \subsection{Configuration options}
                        e.g. "-n", "-np", etc.
 \end{verbatim}
 
+\item LLVM/Clang Developer Options\\
```

Comment:
It is better to put them in the tool section near profiling tools.

Reply:
Ye - this is a cmake config option and needs to go here.  This section should be comprehensive. I suggest merging now and adding an additional set of notes in the tools section after we have some experience.

Reply:
This is a compiler or tool specific option.
Listing an entry here is fine but the following bulk description must be moved to the tools section.

We were not listing CMake options for profiling tools. If the policy is having all the CMake options listed here. We need to list all of them.

Reply:
I'd like to have an entry as brief as possible. Even not listing the two options here. Just put a ref to the tool section.